dotnet/src/Connectors/Connectors.AzureOpenAI.UnitTests/Extensions/AzureOpenAIKernelBuilderExtensionsTests.cs

```diff
@@ -155,7 +156,7 @@ public void KernelBuilderAddAzureOpenAIAudioToTextAddsValidService(Initializatio
     {
         // Arrange
         var credentials = DelegatedTokenCredential.Create((_, _) => new AccessToken());
-        var client = new AzureOpenAIClient(new Uri("https://endpoint"), "key");
+        var client = new AzureOpenAIClient(new Uri("http://endpoint"), new ApiKeyCredential("key"));
```

Comment:
Keep the HTTPS tests in place. Is good to have tests on both.

```suggestion
        var client = new AzureOpenAIClient(new Uri("https://endpoint"), new ApiKeyCredential("key"));
```

Reply:
Huh, wonder how that snuck in, good spot 🤔